.travis.yml

```diff
@@ -57,6 +59,10 @@ after_success:
       docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
       docker push $DOCKER_IMAGE:$DOCKER_TAG;
     fi
+  - if [ "$DOCKER_TAG" == "master" ]; then
```

Comment:
The image structure we are aiming for:
- tagged releases, with dockertag = git-tag
- head of each branch (i.e. moving targer), with dockertag = git branch name
- PRs, with dockertag = pr-<targetbranch>-<id>
- dockertag latest = to dockertag master.
This is for the last point.